client/src/main/java/io/getstream/chat/android/client/notifications/ChatNotifications.kt

```diff
@@ -135,7 +134,7 @@ internal class ChatNotifications private constructor(
 
         val notification = handler.buildNotification(
             notificationId,
-            channel.extraData.getOr("name", "").toString(),
+            channel.extraData["name"].toString(),
```

Comment:
Is it safe to do this without a `getOrDefault` here, are we sure we'll have a value in the Map?

Reply:
Good catch.
Don't know why calling getOrDefault() results in "Call requires API level 24", weird 🤔. 
Refactored using elvis op.

Reply:
It's because [that method in `Map`](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#getOrDefault-java.lang.Object-V-) was introduced in JDK8. We do have [`getOrElse`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/get-or-else.html) available though, which I actually meant to suggest, I just mixed up the two names 😅 .

The Elvis solution seems good too though.